data/json/items/comestibles/raw_grain.json

```diff
@@ -0,0 +1,356 @@
+    "name": { "str_sp": "dried oat stalks" },
+    "copy-from": "dry_wheat_stalks",
+    "quench": -5,
+    "calories": 355,
+    "description": "A sheaf of dried oat stalks.  These will require threshing before anything can be done with them.",
+    "price": 12,
+    "price_postapoc": 10
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "threshed_oats",
+    "name": { "str_sp": "threshed oats" },
+    "copy-from": "threshed_wheat",
+    "quench": -5,
+    "calories": 340,
+    "description": "Oats with the chaff still mixed in.  They will have to be winnowed before they're usable.",
+    "price": 12,
+    "price_postapoc": 10
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "oats",
+    "name": { "str_sp": "oats" },
+    "weight": "203 g",
+    "color": "brown",
+    "comestible_type": "FOOD",
+    "symbol": ",",
+    "quench": -5,
+    "calories": 335,
+    "description": "A pile of raw oats.",
+    "price": 120,
+    "price_postapoc": 100,
+    "material": [ "wheat" ],
+    "volume": "250 ml",
+    "milling": { "into": "flour_wheat_free", "conversion_rate": 3.75 },
+    "vitamins": [ [ "calcium", 3 ], [ "iron", 20 ] ],
+    "petfood": [ "CATTLEFOOD", "BIRDFOOD" ],
+    "flags": [ "EDIBLE_FROZEN", "RAW", "INEDIBLE", "CATTLE", "NUTRIENT_OVERRIDE", "PLANTABLE_SEED", "CUT_HARVEST" ],
+    "seed_data": { "plant_name": "oats", "fruit": "oat_stalks", "grow": "65 days" }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "wild_rice_stalks",
+    "name": { "str_sp": "wild rice stalks" },
+    "copy-from": "buckwheat_stalks",
+    "quench": -5,
+    "calories": 135,
+    "description": "A sheaf of wild rice stalks, ready for drying.",
+    "price": 12,
+    "price_postapoc": 10,
+    "smoking_result": "dry_wild_rice_stalks"
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "dry_wild_rice_stalks",
+    "name": { "str_sp": "dried wild rice stalks" },
+    "copy-from": "dry_buckwheat_stalks",
+    "quench": -5,
+    "calories": 135,
+    "description": "A bunch of dry wild rice stalks.  Threshing these would knock the grains loose.",
+    "price": 12,
+    "price_postapoc": 10
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "wild_rice",
+    "name": { "str_sp": "wild rice" },
+    "copy-from": "oats",
+    "calories": 120,
+    "description": "Raw wild rice, still in its fibrous shell.  It'll need to be milled before it's edible for humans.",
+    "milling": { "into": "dry_wild_rice", "conversion_rate": 5 },
+    "fun": -20,
+    "petfood": [ "CATTLEFOOD", "BIRDFOOD" ],
+    "flags": [ "EDIBLE_FROZEN", "RAW", "INEDIBLE", "CATTLE", "PLANTABLE_SEED", "CUT_HARVEST" ],
+    "seed_data": { "plant_name": "wild rice", "fruit": "wild_rice_stalks", "grow": "91 days" }
+  },
+  {
+    "type": "COMESTIBLE",
+    "id": "canola_stalks",
+    "name": { "str_sp": "canola stalks" },
+    "copy-from": "wheat_stalks",
+    "weight": "171 g",
+    "quench": -5,
+    "calories": 247,
+    "description": "A sheaf of tall grass topped with yellow flowers, now ready for drying.",
```

Comment:
I don't think canola stalks have flowers anymore when they are harvested for seeds.